index.php

```diff
@@ -0,0 +1,44 @@
+<?php
+    $db = new PDO('mysql:host=db; dbname=collection', 'root', 'password');
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    $query = $db->prepare("SELECT * FROM `playing_cards`;");
+    $query->execute();
+    $cardSets = $query->fetchAll();
```

Comment:
Suggest moving these two a function to get all cards as well, again to make this code re-usable.